packages/web-client/src/helpers/share/role.ts

```diff
@@ -19,14 +20,16 @@ export abstract class ShareRole {
     label: string,
     inlineLabel: string,
     icon: string,
-    permissions: SharePermission[]
+    permissions: SharePermission[],
+    labelPrependText?: string,
   ) {
     this._name = name
     this._folder = folder
     this._label = label
     this._inlineLabel = inlineLabel
     this._icon = icon
     this._permissions = permissions
+    this._labelPrependText = labelPrependText || ''
```

Comment:
Please don't define a prefix, but the full label! String concatenation always calls for trouble in translation (think of left to right languages etc). I.e. `_longLabel` with e.g. value `Anyone with the link can view`.